Sources/PaymentButtons/PayPalCreditButton.swift

```diff
@@ -27,7 +27,7 @@ public final class PayPalCreditButton: PaymentButton {
         insets: NSDirectionalEdgeInsets? = nil,
         color: Color = .darkBlue,
         edges: PaymentButtonEdges = .softEdges,
-        size: PaymentButtonSize = .collapsed
+        size: PaymentButtonSize = .regular
```

Comment:
We will need to update the default in both of the init docstrings in this file as well